src/shared/components/FlowPlayer/FlowPlayer.tsx

```diff
@@ -1,18 +1,39 @@
-import React, { FunctionComponent, useEffect, useRef } from 'react';
+import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
+
+import { Icon } from '@viaa/avo2-components';
+import { CustomWindow } from '../../../shared/types/CustomWindow';
+
+import './FlowPlayer.scss';
+
 declare const flowplayer: any;
 
 interface FlowPlayerProps {
-	src: string;
+	src: string | null;
 	poster: string;
+	logo?: string;
+	title: string;
+	start?: number;
+	end?: number;
+	onInit?: () => void;
 }
 
-export const FlowPlayer: FunctionComponent<FlowPlayerProps> = props => {
+export const FlowPlayer: FunctionComponent<FlowPlayerProps> = ({ src, poster, title, onInit }) => {
 	const videoPlayerRef = useRef(null);
 	let player: any = null;
```

Comment:
Linter warning: `Assignments to the 'player' variable from inside React Hook useEffect will be lost after each render. To preserve the value over time, store it in a useRef Hook and keep the mutable value in the '.current' property. Otherwise, you can move this variable directly inside useEffect`